scripts/exgdas_enkf_fcst.sh

```diff
@@ -108,7 +108,9 @@ export LEVS=${LEVS_ENKF:-${LEVS:-64}}
 
 # nggps_diag_nml
 export FHOUT=${FHOUT_ENKF:-3}
-
+if [[ $CDUMP == "gfs" ]]; then
+    export FHOUT=${FHOUT_ENKF_GFS:-${FHOUT_ENKF:${FHOUT:-3}}}
+fi
```

Comment:
Instead, how about this on the line above?
```suggestion
export FHOUT=${FHOUT_ENKF_${CDUMP}:-${FHOUT_ENKF:-3}}
```

Reply:
$CDUMP should be "gfs", not "GFS", should I change FHOUT_ENKF_GFS to FHOUT_ENKF_gfs?

Reply:
That makes sense.

Reply:
To be consistent, I think using FHOUT_ENKF_GFS is better than FHOUT_ENKF_gfs, because we don't have FHOUT_ENKF_GDAS variable. So I think I don't need to change it.